packages/ui-components/src/components/AppShellProvider/AppShellProvider.component.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+
+import { StyleProvider } from "../StyleProvider"
+import { ShadowRoot } from "../ShadowRoot"
+import { PortalProvider } from "../PortalProvider"
+
+const Wrapper = ({ children, shadowRoot, shadowRootMode }: WrapperProps) => {
+  return shadowRoot ? (
+    <ShadowRoot mode={shadowRootMode}>
+      <>{children}</>
+    </ShadowRoot>
+  ) : (
+    children
```

Comment:
I was wondering here if `children` need to be wrapped in a valid JSX tag https://github.com/cloudoperators/juno/pull/547#discussion_r1820773192